server/src/main/java/org/elasticsearch/index/mapper/TimeSeriesParams.java

```diff
@@ -25,18 +25,34 @@ private TimeSeriesParams() {}
 
     public enum MetricType {
         GAUGE(new String[] { "max", "min", "value_count", "sum" }),
-        COUNTER(new String[] { "last_value" });
+        COUNTER(new String[] { "last_value" }),
+        POSITION(new String[] { "last_value", "geo_line" }, false);
 
         private final String[] supportedAggs;
+        private final boolean scalar;
```

Comment:
Not sure about this scalar flag as it is only used in test. I think it is ok but could we add more javadocs of the purpose of it?